src/controllers/authenticationController.ts

```diff
@@ -113,6 +115,16 @@ export default class AuthenticationController {
         : getProfileResponse;
     }
 
+    const deleteInstallationResponse = await this.notificationService.deleteInstallation(
+      user.fiscal_code
```

Comment:
isn't this `toFiscalCodeHash(user.fiscal_code)` ?

Reply:
The encoding happen inside NotificationService.